zerver/lib/webhooks/git.py

```diff
@@ -275,6 +277,18 @@ def get_commits_content(commits_data: List[Dict[str, Any]], is_truncated: Option
         ).replace('  ', ' ')
     return commits_content.rstrip()
 
+def get_release_event_message(user_name: str, action: str,
+                              tagname: str, release_name: str, url: str) -> str:
+    content = RELEASE_MESSAGE_TEMPLATE.format(
+        user_name=user_name,
+        action=action,
+        tagname=tagname,
+        release_name=release_name,
+        url=url
+    )
+
+    return content
```

Comment:
Can you also add a commit refactoring the existing GitHub integration to use this function as well?  

I think while we're thinking about this, we should change the template; I'd think that it should be
`RELEASE_MESSAGE_TEMPLATE = "{user_name} {action} release [{release_name}]({url}) for tag {tagname}"` as to where the link sits

Does that feel more readable to you?

Reply:
(I'd do the GitHub change to use this as a second commit; you can squash the formatting changes to this into the current/first commit).

Reply:
I've implemented the proposed changes. Let me know how it looks!